package.json

```diff
@@ -19,11 +19,12 @@
     "prepare": "husky install",
     "pre-commit": "lint-staged",
     "docs:serve": "rm -rf docs && typedoc --out docs src/**/**/* && http-server docs",
-    "landing:serve": "http-server ./landing_page"
+    "landing:serve": "http-server ./landing_page",
+    "check-ant-changelog": "NODE_OPTIONS=\"--import=./register.mjs\" node ./src/tools/check-ant-changelog.ts"
```

Comment:
don't see this tool in this PR - what is it doing?

Reply:
Oh, I forgot to remove this. The intent was to compare the changelog with the ID we set for source code to ensure the changelog was up to date. I ran into issues running the tool and not being able to import the ID from the constants folder, so I switched over to a simpler jest test and then ran into issues not being able to load the raw changelog as a string there. So I gave up on it for now - will remove the script command.